src/Fmt_ast.ml

```diff
@@ -1420,9 +1428,9 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
           ( wrap_if parens "(" ")"
               (hovbox 0
                  ( hovbox 2
-                     ( hovbox 2
+                     ( hvbox_if_sparse c.conf.fun_call 2
                          ( fmt_args_grouped e0 e1N $ fmt "@ "
-                         $ fmt_label lbl ":@,"
+                         $ fmt_label lbl ":"
```

Comment:
Does reverting this change resolve the issue your comment points to?

Reply:
```diff
-  ; field "id" ~doc:"Toy ID." ~args:[] ~typ:(non_null guid)
-      ~resolve:(function
+  ; field "id" ~doc:"Toy ID." ~args:[] ~typ:(non_null guid) ~resolve:
+      (function
       | A -> x.id
       | B -> c )
```

Reply:
I prefer the version without the break

Reply:
To me that looks as expected. Maybe you still want more indentation though.

Reply:
I wonder if you would rather disable the special case when the last arg of an apply is a function? Perhaps try commenting out lines 1448-1490, maybe even from 1421, and see if that is preferred. Basically the main point of those special cases is to reduct the indentation when there is a lambda as the last arg of an apply. This is useful for e.g. heavily CPSed code, but doesn't make a huge difference most of the time. The idea is that if it is known to be the last arg, then the indentation of the cases can be less since there isn't a possible confusion between the last case and the next arg.

Reply:
I don't want to give up the special case.
We rely heavily on it when using `List.map` &co